stock_warehouse_relationship/tests/test_warehouse_relationship.py

```diff
@@ -0,0 +1,184 @@
+# Copyright 2022 Foodles (http://www.foodles.co).
```

Comment:
this file should probably import from the __init__.py file above unless practices changes in version 16.0 (but I belives I've probably forget it in former PR as long I'm using pytest to launch them